presets/4.3/vtx/speedybee_tx800.txt

```diff
@@ -0,0 +1,37 @@
+#$ TITLE: SpeedyBee TX800 VTX Tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, speedybee, tx800
+#$ AUTHOR: NemesisXB
+#$ DESCRIPTION: VTX tables for the SpeedyBee TX800.
+#$ DESCRIPTION: Select the regulatory domain in the options (EU or US).
+#$ DISCLAIMER:  Only select one regulatory option. All previous VTX Table settings will be reset.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/105
+
+#$ INCLUDE: presets/4.3/vtx/defaults_vtx_tables.txt
+
+# Configure common vtxtable settings
+vtxtable bands 5
+vtxtable channels 8
+vtxtable powerlevels 5
+vtxtable powervalues 25 200 400 600 600
+vtxtable powerlabels 25 200 400 800 800
```

Comment:
same here... looks like Diatone messed things up 0_o

Reply:
This one just says Tramp as VTX Type is the Video Transmitter page. For this I can probably drop the power levels to just 4 as the last one is duplicate. But thought I would keep it the same as there json file. Do you need me to change that as well? Or are you happy with this as is?

Reply:
@limonspb, I tested this one by changing the powervalues to match the powerlabels and it stopped working.

I believe this should be merged as is. This VTX also uses IRC Tramp, do you want me to add (IRC Tramp) to the name of the preset and the description? Tramp seems to not be versioned (I assume there is only 1 version so far), so I do not see any reason to do that though.

Reply:
@NemesisXB  good job yes!